src/attrcache.c

```diff
@@ -234,7 +234,6 @@ int git_attr_cache__get(
 		/* remove existing entry */
 		if (file) {
 			attr_cache_remove(cache, file);
-			git_attr_file__free(file); /* offset incref from lookup */
```

Comment:
This is a line I'm always stumbling upon every few weeks and I'm still not really sure what is the right thing to do here. As-is, this creates a memory leak with attr files, so this is not the correct thing to do. Locally, I'm running with an assert here that file still has a positive reference-count, and until now it did not trigger. And I think this is also expected, as the comment also states that this is to offset the lookup-incref.

So I'm not really sure what to do here. I'd guess this is simply a false positive of Coverity.